### PR TITLE
TF-359 Open a link in email

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/161949082-60b91bc9-f4da-4ba1-b6f7-a031cd34ff7c.mov

### CODE REVIEW

File: core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart [hunk 1]
```diff
@@ -106,7 +110,7 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
             return false;  
           }
         
-          return url.protocol === "http:" || url.protocol === "https:";
+          return url.protocol === "http:" || url.protocol === "https:" || url.protocol === "mailto:";
```
Comment (hoangdat) [inline core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:113]: IMO you should change the method name to: `isValidUrl`

**Comment by hoangdat** [comment]
image

I will config it and update it soon.

File: core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart (hunk 1 above)
Comment (dab246) [reply core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart:113]: ok

**Review by hoangdat — APPROVED** [review]

**Comment by hoangdat** [comment]
I merge it for our DevOps engineer working on it, CI/CD will be off for a while

**Pull request merged**